server.js

```diff
@@ -1,47 +1,43 @@
-require('dotenv').config();
-const express = require("express");
-const app = express();
-const { connectDB } = require("./config/database");
-const indexRouter = require("./routes/index");
-const authRouter = require("./routes/auth");
-const adminRouter = require("./routes/admin");
-const productRouter = require("./routes/product");
-const userRouter = require("./routes/user");
-const cors = require("cors");
-const bodyParser = require("body-parser");
-//const { JWT } = require("google-auth-library");
-const paystackRouter = require("./routes/paystack");
-const NotFound = require('./middlewares/notFound');
-const errorHandler = require('./middlewares/errorHandler');
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const { connectDB } = require('./config/database')
+const indexRouter = require('./routes/index')
+const authRouter = require('./routes/auth')
+const adminRouter = require('./routes/admin')
+const productRouter = require('./routes/product')
+const userRouter = require('./routes/user')
+const cors = require('cors')
+const bodyParser = require('body-parser')
+const paystackRouter = require('./routes/paystack')
+const NotFound = require('./middlewares/notFound')
+const errorHandler = require('./middlewares/errorHandler')
 
-app.use(express.json());
-app.use(express.urlencoded({ extended: true }));
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(cors());
+app.use(express.json())
+app.use(express.urlencoded({ extended: true }))
+app.use(bodyParser.urlencoded({ extended: true }))
 
+app.use(cors())
+app.use('/api/v1/', indexRouter)
+app.use('/api/v1/auth/admin', adminRouter)
+app.use('/api/v1/users', userRouter)
+app.use('/api/v1/auth', authRouter)
+app.use('/api/v1/products', productRouter)
+app.use('/api/v1/paystack', paystackRouter)
 
-app.use("/api/v1/", indexRouter);
-app.use("/api/v1/auth/admin", adminRouter);
-app.use("/api/v1/users", userRouter);
-app.use("/api/v1/auth", authRouter);
-app.use("/api/v1/products", productRouter);
-app.use("/api/v1/paystack", paystackRouter);
-app.use("/api/v1/public/uploads/productImages", express.static("public/uploads/productImages"))
```

Comment:
This is the path i refer to instead of "req.file.path", again i don't know about multer-cloudinary so i would like to see what's contained in req.file.path

Reply:
i know, i removed it because it wasnt needed anymore. its coming  from a url so there's no need..